### PR TITLE
dotnet watch to msbuild

### LINKED ISSUE #2727: aspnetcore/tutorials/dotnet-watch.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
fixes #2727

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra  please review.

File: aspnetcore/tutorials/dotnet-watch.md [hunk 1]
```diff
@@ -1,195 +1,98 @@
...
-# Developing ASP.NET Core applications using dotnet watch
+# Developing ASP.NET Core apps using dotnet watch
 
 <a name=dotnet-watch></a>
 
-By [Victor Hurdugaci](https://twitter.com/victorhurdugaci)
+By [Victor Hurdugaci](https://twitter.com/victorhurdugaci) and [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-## Introduction
+`dotnet watch` is a tool that runs a `dotnet` command when source files change.  For example, when a file changes it can compile your code, run tests, or publish your app.
```
Comment (tdykstra) [inline aspnetcore/tutorials/dotnet-watch.md]: or: 
For example, a file change can trigger compilation, tests, or deployment.

File: aspnetcore/tutorials/dotnet-watch.md [hunk 2]
```diff
@@ -1,195 +1,98 @@
...
 
-`dotnet watch` is a development time tool that runs a `dotnet` command when source files change. It can be used to compile, run tests, or publish when code changes.
+In this tutorial we'll use an existing Web API app with two endpoints; one that returns a sum and one that returns a product. The product method contains an intentional bug that we'll fix as part of this tutorial.
```
Comment (tdykstra) [inline aspnetcore/tutorials/dotnet-watch.md]: we --> you    throughout the tutorial for consistency
colon after "endpoints"
~~intentional~~

File: aspnetcore/tutorials/dotnet-watch.md [hunk 3]
```diff
@@ -1,195 +1,98 @@
...
 
-In this tutorial we'll use an existing WebApi application that calculates the sum and product of two numbers to demonstrate the use cases of `dotnet watch`. The sample application contains an intentional bug that we'll fix as part of this tutorial.
+Download the [sample app](https://github.com/aspnet/Docs/tree/master/aspnetcore/tutorials/dotnet-watch/sample). It contains two projects, `WebApp` (a web app) and `WebAppTests` (unit tests for the web app).
```
Comment (tdykstra) [inline aspnetcore/tutorials/dotnet-watch.md]: Given that this tutorial depends on the reader doing a download, might be worth adding instructions (i.e., download the entire repo and navigate to the right folder)

File: aspnetcore/tutorials/dotnet-watch.md [hunk 4]
```diff
@@ -1,195 +1,98 @@
...
-In a console, open the folder where you downloaded the sample application and run:
-
-1. `dotnet restore`
-2. `cd WebApp`
-3. `dotnet run`
-
-The console output will show messages similar to the ones below, indicating that the application is now running and waiting for requests:
+The console output will show messages similar to the following (indicating that the app is running and waiting for requests):
 
 ```bash
```
Comment (tdykstra) [inline aspnetcore/tutorials/dotnet-watch.md]: console rather than bash since the text is saying console

**Pull request merged**